README.md

```diff
@@ -178,16 +178,22 @@ In your project directory, run `cargo libcnb package` to start packaging:
 
 ```shell
 $ cargo libcnb package
-INFO - Reading buildpack metadata...
-INFO - Found buildpack libcnb-examples/my-buildpack with version 0.1.0.
-INFO - Determining automatic cross-compile settings...
-INFO - Building binaries (x86_64-unknown-linux-musl)...
+🔍 Locating buildpacks...
+📦 [1/1] Building libcnb-examples/my-buildpack
+Determining automatic cross-compile settings...
+Building binaries (x86_64-unknown-linux-musl)...
 # Omitting compilation output...
-    Finished dev [unoptimized] target(s) in 8.51s
-INFO - Writing buildpack directory...
-INFO - Successfully wrote buildpack directory: target/buildpack/debug/libcnb-examples_my-buildpack (3.03 MiB)
-INFO - Packaging successfully finished!
-INFO - Hint: To test your buildpack locally with pack, run: pack build my-image --buildpack target/buildpack/debug/libcnb-examples_my-buildpack --path /path/to/application
+    Finished dev [unoptimized] target(s) in 8.92s
+Writing buildpack directory...
+Successfully wrote buildpack directory: packaged/x86_64-unknown-linux-musl/debug/libcnb-examples_my-buildpack (4.06 MiB)
```

Comment:
I'm curious why the buildpack size has increased by 30% since the last time this was refreshed?

Reply:
I saw that too and knew you would latch onto it immediately :) It could really be anything since this is a debug build - rustc might have added more debug symbols or, more likely, we added more dependencies since we wrote the guide. I think I remember writing it before most of us even had CNBs for their languages.

FWIW: I re-ran this with `--release` and the size dropped to 1.78 MiB.

Reply:
The guide was last refreshed for libcnb 0.11.2, so not that long ago. Agree it might be debug-only inflation, since some new transitive but not really used dep might not get pruned in debug mode. Looking at the diff there were also some `toml` and `fancy-regex` major version upgrades - I know both projects have had significant overhauls (eg toml had a refactor due to wanting to support editing too etc).

Reply:
I saw increases in Procfile CNB release builds too - which I looked into here:
https://github.com/heroku/procfile-cnb/pull/175#issuecomment-1686312268

tl;dr: Mostly regex + toml dependency changes. We should also stop using regex when we don't need to use regex.